pep-0517.txt

```diff
@@ -166,48 +166,71 @@ The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
 argument is described after the individual hooks::
 
-  def get_build_requires(config_settings):
+  def build_sdist(source_directory, sdist_directory, config_settings):
+      ...
+
+MUST build an unpacked sdist from the ``source_directory`` and place it into the
```

Comment:
> Do we really want to leave it to the backend to decide what goes into a sdist?

As of now, this is the status quo. Some dists even require plugins (e.g. `setuptools-git`) so their backends can do that correctly.

> Certainly the backend needs to say "we must have these files if we are to be able to build a wheel",

Which is why Thomas would prefer if we restricted this method to only handle the files needed to build the wheel, and tabling the whole `sdist` discussion for later.

> I think I'd prefer a `get_build_files` hook that has to return a list of the files needed for a build

Not sure if it would be better to list the files instead of copying them, as a backend like `flit` could satisfy the current method with a `shutil.copytree()`. Maybe we just need to rename the method? The only thing that really needs to happen is for the destination directory to contain something, created by the backend, that the backend itself knows how to turn it into a wheel